### PR TITLE
Docs: add note on permalink

### PR DESCRIPTION
**Pull request by sukrosono**
this commit intended to close hexo/#4663 .
cheers, happy coding :)

## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
  - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [x] `en` English
  - [ ] `ko` Korean
  - [ ] `pt-br` Brazilian Portuguese
  - [ ] `ru` Russian
  - [ ] `th` Thai
  - [ ] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

File: source/docs/front-matter.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Setting | Description | Default
 `comments` | Enables comment feature for the post | true
 `tags` | Tags (Not available for pages) |
 `categories` | Categories (Not available for pages) |
-`permalink` | Overrides the default permalink of the post |
+`permalink` | Overrides the default permalink of the post | `null` but keep remember permalink sould end with `/` or `.html`
```
Comment (SukkaW) [inline source/docs/front-matter.md:37]:
```suggestion
`permalink` | Overrides the default permalink of the post. Permalink sould end with `/` or `.html` | `null`
```

**Review by SukkaW — CHANGES_REQUESTED** [review]

**Comment by SukkaW** [comment]
Also, would you mind adding this to all languages? Just copy the sentence in English to every languages' markdown file and will be fine.

**Comment by sukrosono** [comment]
eh, my priority changes then. It should have my nationality languages too [id]. Maybe next year, but don't close this,

**Review by SukkaW — APPROVED** [review]

**Review by yoshinorin — APPROVED** [review]
Thank you :)

**Pull request merged**